pylua/timer.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+
+"""
+Module implements wrapper of python
+timer. Wrapper allows to use timer with milliseconds
+"""
+
+import threading
+
+
+def msec_to_sec(milliseconds):
+    milliseconds_in_second = 1000.0
+    return milliseconds/milliseconds_in_second
```

Comment:
@dtrunov, please handle none param value in both helpers. I think assert is enough